clang/lib/AST/RecordLayoutBuilder.cpp

```diff
@@ -2866,8 +2871,10 @@ MicrosoftRecordLayoutBuilder::layoutNonVirtualBases(const CXXRecordDecl *RD) {
   for (const CXXBaseSpecifier &Base : RD->bases()) {
     if (Base.isVirtual())
       continue;
+
     const CXXRecordDecl *BaseDecl = Base.getType()->getAsCXXRecordDecl();
     const ASTRecordLayout &BaseLayout = Context.getASTRecordLayout(BaseDecl);
+
```

Comment:
Nit: After adding and removing things here, the total result of the PR ends up in adding new spurious empty lines here (and right above, and below on line 2949 and 2967).